src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/CSharpWorkspaceExtensions.projitems

```diff
@@ -53,6 +53,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)LanguageServices\CSharpRemoveUnnecessaryImportsService.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)LanguageServices\CSharpRemoveUnnecessaryImportsService.Rewriter.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)LanguageServices\CSharpReplaceDiscardDeclarationsWithAssignmentsService.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Lightup\NullableSyntaxAnnotationEx.cs" />
```

Comment:
This file will compile even into C# Workspaces layer, which should never require lightup. Did you mean to instead include it directly into CodeStyle C# CodeFixes project?

Reply:
Yeah, I hate lightup/reflection code, and now making such code available in Workspaces layer is code smell. I think we need a better solution as mentioned in https://github.com/dotnet/roslyn/pull/46873#discussion_r471571746. Cleaning up `#if CODE_STYLE` directives by adding shared lightup/reflection code does not seem a good path forward. Lets discuss this at team meeting.